Dockerfile

```diff
@@ -17,12 +17,10 @@ COPY vendor/ vendor/
 RUN CGO_ENABLED=0 GO111MODULE=on go build -mod=vendor -a -o manager main.go \
   && CGO_ENABLED=0 GO111MODULE=on go build -mod=vendor -a -o daemon ./cmd/daemon/main.go
 
-# Use Ubuntu 20.04 LTS as base image to package the binaries
-FROM ubuntu:focal
-# This is required by daemon connnecting with CRI
-RUN ln -s /usr/bin/* /usr/sbin/ && apt-get update -y \
-  && apt-get install --no-install-recommends -y ca-certificates \
-  && apt-get clean && rm -rf /var/log/*log /var/lib/apt/lists/* /var/log/apt/* /var/lib/dpkg/*-old /var/cache/debconf/*-old
+FROM alpine:3.17
```

Comment:
I suggest to use 
```
ARG BASE_IMAGE
```
and set its default value is alpine:3.17, then user could customize the base image.